mist/runtime/loss.py

```diff
@@ -1,7 +1,7 @@
 import torch
 import torch.nn as nn
 from torch.nn.functional import softmax, one_hot
-
+from .soft_skeleton import SoftSkeletonize
```

Comment:
This should be the following

from mist.runtime.soft_skeleton import SoftSkeletonize